pallets/elections/src/lib.rs

```diff
@@ -73,12 +92,23 @@ pub mod pallet {
         type SessionInfoProvider: SessionInfoProvider<Self>;
         /// Something that handles addition of rewards for validators.
         type ValidatorRewardsHandler: ValidatorRewardsHandler<Self>;
+
+        /// Maximum acceptable kick-out reason length.
+        #[pallet::constant]
+        type MaximumKickOutReasonLength: Get<u32>;
     }
 
     #[pallet::event]
     #[pallet::generate_deposit(pub(super) fn deposit_event)]
     pub enum Event<T: Config> {
+        /// Emitted via [`Pallet<T>::change_validators`]
         ChangeValidators(Vec<T::AccountId>, Vec<T::AccountId>, CommitteeSeats),
+
+        /// Emitted via [`Pallet<T>::set_kick_out_thresholds`]
+        SetCommitteeKickOutThresholds(CommitteeKickOutThresholds),
+
+        /// Emitted during elections when there are some underperforming validators
+        KickOutValidators(BTreeSet<T::AccountId>),
```

Comment:
Hmm, why not `Vec` instead of `BTreeSet`?

Reply:
This was due to the fact in the code where this is emitted I have already `BTreeSet` and I want to avoid another O(N) iteration. On the UI side, it looks very similar, let me get the screenshot.

Reply:
Ok, fine, let's leave it then.

Reply:
![image](https://user-images.githubusercontent.com/3909333/194017085-c3a66519-038e-4c65-8546-9b841cd4f199.png)